src/unistd.rs

```diff
@@ -214,6 +214,36 @@ pub fn write(fd: RawFd, buf: &[u8]) -> Result<usize> {
     Errno::result(res).map(|r| r as usize)
 }
 
+pub enum Whence {
+    SeekSet,
+    SeekCur,
+    SeekEnd
```

Comment:
If I use the constants from libc, which do you think it's the best place to add these two ?